### PR TITLE
Add support for external tickets company details

### PR DESCRIPTION
**Pull request by TimvdLippe**
Since this is extra necessary information and there is 1 external ticket sold. We still need to contact this person with the details, since this PR was not open before :sob:

### CODE REVIEW

File: src/symposium-tickets/symposium-tickets-button.html [hunk 1]
```diff
@@ -77,18 +80,26 @@
...
           name: name,
           email: email
         };
+        var companyName = this.parentNode.querySelector('input[name="name"]').value;
+        var companyPostalAddress = this.parentNode.querySelector('input[name="companypostaladdress"]').value;
+        if (selectedTicket.value === 'External') {
+          if (!(companyName && companyPostalAddress)) {
+            this.$.errorMessage.style.display = 'block';
+            return;
+          }
```
Comment (martijnjanssen) [inline src/symposium-tickets/symposium-tickets-button.html:93]: The else block should hide this portion of the page again.

File: src/symposium-tickets/symposium-tickets-button.html [hunk 2]
```diff
@@ -77,18 +80,26 @@
           this.$.errorMessage.style.display = 'block';
           return;
         }
-        this.$.button.textContent = 'Processing...';
-
         var user = {
           name: name,
           email: email
         };
+        var companyName = this.parentNode.querySelector('input[name="name"]').value;
```
Comment (martijnjanssen) [inline src/symposium-tickets/symposium-tickets-button.html:87]: I believe this should be name=companyname

**Review by martijnjanssen — CHANGES_REQUESTED** [review]
Looks like we can improve here :)

**Comment by TimvdLippe** [comment]
I would vouch for closing this one, as it is too risky and too late now. We can improve for next year nonetheless.

**Pull request closed without merging**